test/unit/connectors/ExportsConnectorSpec.scala

```diff
@@ -76,76 +76,76 @@ class MdgExportsConnectorSpec extends UnitSpec with MockitoSugar with BeforeAndA
 
   private implicit val vpr = TestCspValidatedPayloadRequest
 
-  "MdgExportsConnector" can {
+  "ExportsConnector" can {
 
     "when making a successful request" should {
 
-      "use stub service url" in {
+      "ensure URL is retrieved from config" in {
         returnResponseForRequest(Future.successful(mock[HttpResponse]))
 
-        awaitSubscriptionFields
+        awaitRequest
 
         verify(mockWsPost).POSTString(ameq(serviceConfig.url), anyString, any[Seq[(String, String)]])(
           any[HttpReads[HttpResponse]](), any[HeaderCarrier](), any[ExecutionContext])
       }
 
-      "pass the xml in the body" in {
+      "ensure xml payload is included in the request body" in {
         returnResponseForRequest(Future.successful(mock[HttpResponse]))
 
-        awaitSubscriptionFields
+        awaitRequest
 
         verify(mockWsPost).POSTString(anyString, ameq(xml.toString()), any[Seq[(String, String)]])(
           any[HttpReads[HttpResponse]](), any[HeaderCarrier](), any[ExecutionContext])
       }
 
-      "set the content type header" in {
+      "ensure the content type header in passed through in the request" in {
```

Comment:
Typo? "ensure the content type header is passed through in the request"

Reply:
Fixed